### PR TITLE
table molecule created

### PR DESCRIPTION
**Pull request by pranitchandel**

### CODE REVIEW

File: src/shared/molecules/Table/index.tsx [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+const Table = (props: tablePropType) => {
+  const { checkboxSelection, rows, columns, rowsPerPageOptions, pageSize } =
+    props;
+
+  const [newPageSize, setNewPageSize] = useState<number>(
+    pageSize !== undefined ? pageSize : -1
+  );
+
+  return (
+    <div style={{ height: 400, width: "100%" }}>
```
Comment (AbhijeetKumarGupta) [inline src/shared/molecules/Table/index.tsx:21]: please pass this style as a prop instead of hard coding it

**Review by AbhijeetKumarGupta — COMMENTED** [review]
please resolve the comment

File: src/pages/OverviewPage/InputFieldsSection/index.tsx [hunk 2]
```diff
@@ -1,6 +1,7 @@
 import { Formik } from "formik";
 import TreeSelect from "../../../shared/atom/Inputs/TreeInput";
 import FileUploadInput from "../../../shared/atom/Inputs/FileUploadInput";
+import Table from "../../../shared/molecules/Table";
```
Comment (lakDew) [inline src/pages/OverviewPage/InputFieldsSection/index.tsx:4]: isko theek kr be

**Review by lakDew — APPROVED** [review]

**Review by lakDew — COMMENTED** [review]
dhang se bana

File: src/shared/molecules/Table/index.tsx (hunk 1 above)
Comment (pranitchandel) [reply src/shared/molecules/Table/index.tsx]: done

File: src/pages/OverviewPage/InputFieldsSection/index.tsx (hunk 2 above)
Comment (pranitchandel) [reply src/pages/OverviewPage/InputFieldsSection/index.tsx:4]: plz don't spam here.

**Review by lakDew — COMMENTED** [review]
chlo sutta marte hai

**Review by lakDew — APPROVED** [review]

**Pull request merged**